cli/compose/loader/loader_test.go

```diff
@@ -825,3 +825,31 @@ networks:
 
 	assert.Equal(t, expected, config.Networks)
 }
+
+func TestLoadExpandedMountFormat(t *testing.T) {
+	config, err := loadYAML(`
+version: "3.1"
+services:
+  web:
+    image: busybox
+    volumes:
```

Comment:
Shouldn't these be called mounts? It is very confusing to have a volume declaration here then a volume declaration at the top level. This contributes to confusion. Not that this section will also handle binds, so calling it volumes is inaccurate.